### PR TITLE
Create 0028-find-the-index-of-the-first-occurence-in-a-string

### PR DESCRIPTION
**Pull request by benmak11**
[//]: # "Pull Request Template"
[//]: # "Replace the placeholder values in the template below"

- **File(s) Modified**: _0028-find-the-index-of-the-first-occurrence-in-a-string.java_
- **Language(s) Used**: _java_
- **Submission URL**: _https://leetcode.com/problems/find-the-index-of-the-first-occurrence-in-a-string/submissions/910556174/_

[//]: # "Getting the Submission URL"
[//]: # "Go to the leetcode [`Submissions tab`](https://user-images.githubusercontent.com/71089234/180188604-b1ecaf90-bf27-4fd6-a559-5567aebf8930.png)"
[//]: # "and [click on the `Accepted` status of your submission.](https://user-images.githubusercontent.com/71089234/180189321-1a48c33f-aa65-4b29-8aaa-685f4f5f8c9e.png)]"
[//]: # "Finally copy the URL from the nav bar, it should look like https://leetcode.com/problems/[problem-name]/submissions/xxxxxxxxx/"

### Important
Please make sure the file name is lowercase and a duplicate file does not already exist before merging.

### CODE REVIEW

File: java/0028-find-the-index-of-the-first-occurrence-in-a-string.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        if (haystack.length() <= 0 && needle.length() > 0) return -1;
+        if (haystack.length() != 0) {
+            int occurence = haystack.indexOf(needle);
+            if (occurence == -1) 
+                return occurence;
+            result = occurence;
+        }
+        return result;
+    }
+}
```
Comment (benmak11) [inline java/0028-find-the-index-of-the-first-occurrence-in-a-string.java:13]:
```suggestion
}

```

**Comment by a93a** [comment]
Hi! A file with a solution for this already exists, but your solution differs so we could add it too, to the file.

Could you edit the existing file and add your solution to it, and I can merge it.

You can separate the solutions like this:

```
// Solution ONE (you can write the type of yuor solution like "brute-force", "KMP", "Hashing" etc)
class Solution {
    public int strStr(String haystack, String needle) {
        //...
    }
}

// Solution TWO
class Solution {
    public int strStr(String haystack, String needle) {
        //...
    }
}
```

**Comment by benmak11** [comment]
> Hi! A file with a solution for this already exists, but your solution differs so we could add it too, to the file.
> 
> Could you edit the existing file and add your solution to it, and I can merge it.
> 
> You can separate the solutions like this:
> 
> ```
> // Solution ONE (you can write the type of yuor solution like "brute-force", "KMP", "Hashing" etc)
> class Solution {
>     public int strStr(String haystack, String needle) {
>         //...
>     }
> }
> 
> // Solution TWO
> class Solution {
>     public int strStr(String haystack, String needle) {
>         //...
>     }
> }
> ```

Sorry for the delay on this, I have been traveling. I updated the PR based on your recommendation. Please let me know if there's anything else that needs to be done

**Comment by benmak11** [comment]
@a93a sorry to bother you, but have you had a chance to review the changes applied?

**Comment by a93a** [comment]
@benmak11 my apologies for the very late reply, I must somehow missed your update, sorry for that. But it looks good, thanks for your contribution.

**Pull request merged**